src/document-ready.js

```diff
@@ -28,18 +28,18 @@ export function isDocumentReady(doc) {
 /**
  * Calls the callback when document is ready.
  * @param {!Document} doc
- * @param {!Function} callback
+ * @param {function(!Document)} callback
  */
 export function onDocumentReady(doc, callback) {
   let ready = isDocumentReady(doc);
   if (ready) {
-    callback();
+    callback(doc);
```

Comment:
unrelated to your change, but it bothers me that this is synchronous making the promise and callback versions behave differently...